pages/guide.md

```diff
@@ -99,3 +100,19 @@ Browsers that don't support HSTS are simply unaffected by HSTS, so there is no h
 [Pulse](https://pulse.cio.gov) looks for server-side redirects, using an appropriate HTTP response code. Sites that use client-side redirects -- such as a &lt;meta refresh&gt; tag or JavaScript -- will not be seen as redirects. To meet the M-15-13 requirement of enforcing HTTPS, agencies should employ server-side redirects (or alternatively, disable HTTP access altogether).
 
 Sites that are reachable on both a root domain (`http://agency.gov`) and their www subdomain (`http://www.agency.gov`) should perform a redirect to HTTPS in both cases. Redirecting one but not the other could also cause Pulse to indicate that a domain does not enforce HTTPS.
+
+### Are federally operated certificate revocation services (CRL, OCSP) also required to move to HTTPS?
+
+No. This very narrow class of services, that provide CRL and OCSP information for the purposes of verifying the revocation status of certificates used to make other HTTPS connections, should abide by industry best practices and their respective specifications.
```

Comment:
Looks great!

On Thu, Mar 24, 2016 at 6:00 PM, Eric Mill <notifications@github.com> wrote:

> In pages/guide.md
> <https://github.com/GSA/https/pull/187#discussion_r57394897>:
>
> > @@ -99,3 +100,19 @@ Browsers that don't support HSTS are simply unaffected by HSTS, so there is no h
> >  [Pulse](https://pulse.cio.gov) looks for server-side redirects, using an appropriate HTTP response code. Sites that use client-side redirects -- such as a &lt;meta refresh&gt; tag or JavaScript -- will not be seen as redirects. To meet the M-15-13 requirement of enforcing HTTPS, agencies should employ server-side redirects (or alternatively, disable HTTP access altogether).
> >
> >  Sites that are reachable on both a root domain (`http://agency.gov`) and their www subdomain (`http://www.agency.gov`) should perform a redirect to HTTPS in both cases. Redirecting one but not the other could also cause Pulse to indicate that a domain does not enforce HTTPS.
> > +
> > +### Are federally operated certificate revocation services (CRL, OCSP) also required to move to HTTPS?
> > +
> > +No. This very narrow class of services, that provide CRL and OCSP information for the purposes of verifying the revocation status of certificates used to make other HTTPS connections, should abide by industry best practices and their respective specifications.
>
> @alex <https://github.com/alex> What do you think about:
>
> Agencies are encouraged to operate OCSP and CRL services via hostnames
> specifically reserved for those services, so that other related information
> and functionality can be served securely and privately.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GSA/https/pull/187/files/da22f9f580c5f2360f680da0f4cdbad9be2601ab#r57394897>
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6